components/mpas-ocean/bld/build-namelist

```diff
@@ -1446,7 +1450,11 @@ add_default($nl, 'config_AM_timeSeriesStatsDaily_backward_output_offset');
 #############################################
 
 add_default($nl, 'config_AM_timeSeriesStatsMonthly_enable');
-add_default($nl, 'config_AM_timeSeriesStatsMonthly_compute_on_startup');
+if ($CONTINUE_RUN eq 'TRUE') {
+	add_default($nl, 'config_AM_timeSeriesStatsMonthly_compute_on_startup', 'val'=>".false.");
+} else {
+	add_default($nl, 'config_AM_timeSeriesStatsMonthly_compute_on_startup', 'val'=>".true.");
+}
```

Comment:
This and similar changes above and below for other time-series stats seems to be necessary so that the first day or month in an E3SM simulation is complete.  Let me know if you have concerns.